src/Magnum/Platform/EmscriptenApplication.h

```diff
@@ -0,0 +1,1509 @@
+         *      compatibility with other toolkits. You need to set the title
+         *      separately in the `EmscriptenManifest.xml` file.
+         */
+        template<class T> Configuration& setTitle(const T&) { return *this; }
+
+        /** @brief Window size */
+        Vector2i size() const { return _size; }
+
+        /**
+         * @brief Set window size
+         * @param size              Desired window size
+         * @param dpiScaling        Custom DPI scaling value
+         *
+         * Default is a zero vector, meaning a value that matches the display
+         * or canvas size is autodetected. See @ref Platform-Sdl2Application-dpi
+         * for more information.
+         * When @p dpiScaling is not a zero vector, this function sets the DPI
+         * scaling directly. The resulting @ref EmscriptenApplication::windowSize()
+         * is @cpp size*dpiScaling @ce and @ref EmscriptenApplication::dpiScaling()
+         * is @p dpiScaling.
+         */
+        Configuration& setSize(const Vector2i& size, const Vector2& dpiScaling={}) {
+            _size = size;
+            _dpiScaling = dpiScaling;
+            return *this;
+        }
+
+        /**
+         * @brief Custom DPI scaling
+         *
+         * If zero, the devices pixel ratio has a priority over this value.
+         * The `--magnum-dpi-scaling` command-line option has a priority
+         * over any application-set value.
+         * @see @ref setSize(const Vector2i&, const Vector2&)
+         */
+        Vector2 dpiScaling() const { return _dpiScaling; }
+
+        /** @brief Window flags */
+        WindowFlags windowFlags() const {
+            return _windowFlags;
+        }
+
+        /**
+         * @brief Set window flags
+         * @return  Reference to self (for method chaining)
+         *
+         * Default is @ref WindowFlag::Focused.
+         */
+        Configuration& setWindowFlags(WindowFlags windowFlags) {
+            _windowFlags = windowFlags;
+            return *this;
+        }
+
+    private:
+        Vector2i _size;
+        Vector2 _dpiScaling;
+        WindowFlags _windowFlags;
+};
+
+/**
+@brief Viewport event
+
+@see @ref viewportEvent()
+*/
+class EmscriptenApplication::ViewportEvent {
+    public:
+        /** @brief Copying is not allowed */
+        ViewportEvent(const ViewportEvent&) = delete;
+
+        /** @brief Moving is not allowed */
+        ViewportEvent(ViewportEvent&&) = delete;
+
+        /** @brief Copying is not allowed */
+        ViewportEvent& operator=(const ViewportEvent&) = delete;
+
+        /** @brief Moving is not allowed */
+        ViewportEvent& operator=(ViewportEvent&&) = delete;
+
+        /**
+         * @brief Window size
+         *
+         * On some platforms with HiDPI displays, window size can be different
+         * from @ref framebufferSize(). See @ref Platform-Sdl2Application-dpi
+         * for more information.
```

Comment:
Not the case on the web. Should be just `Equivalent to @ref framebufferSize().`, similarly below.

Reply:
Is the mention of it being different on other platforms still be worth keeping?

Reply:
No. This could be misleading people to think that EmscriptenApplication works differently on non-Emscripten platforms, which makes no sense.